### ISSUE TITLE
DataGrid: Incorrect Aria-checked attribute

### ISSUE THREAD

**Issue by jddimaano** [infor-design/enterprise#4491]
**Describe the bug**
When the column header checkbox has a mixed state, its aria-checked attribute is set to "false".

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/datagrid/example-multiselect.html
2. Select first two rows
3. Inspect the DOM on the header checkbox -> see that aria-selected = false in this case should be mixed

**Expected behavior**
When the checkbox is in its mixed state, aria-checked attribute should be set to "mixed".

**Version**

- ids-enterprise: v4.34

**Screenshots**
Please see attached.
![aria-checked](https://user-images.githubusercontent.com/72140687/95911049-870d7a00-0dd3-11eb-8ae1-dcb68035ebfc.jpg)

**Platform**
 - Infor Application/Team Name: [e.g. Infor XYZ, Infor ABC]
 - Device: (if applicable) [e.g. iPhone 6 or Samsung Galaxy S6]
 - OS Version: [e.g. Windows 10 or iOS 8]
 - Browser Name: [e.g. chrome, safari, stock browser]
 - Browser Version: [e.g. 22, 66.0.3359.181 (Official Build) (64-bit)]

**Additional context**
[https://jira.lawson.com/browse/LMCLIENT-32565](url)

**Comment by janahintal** [comment]
Failing in this scenario:

1. select one (not all rows)
2. inspect the header checkbox dom and the aria-checked state **should** be **mixed**

Actual: aria-checked = false
![image](https://user-images.githubusercontent.com/45757016/102746098-80355180-4398-11eb-9a8f-c7a65030062f.png)

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
